### PR TITLE
Shivam/seo

### PR DESCRIPTION
**Pull request by shivamag00**

### CODE REVIEW

File: src/views/TagsList.vue [hunk 1]
```diff
@@ -27,9 +27,17 @@ export default {
...
   mounted() {
     this.fetchTaglist();
+      const titleEl = document.querySelector('head title');
+      titleEl.textContent = "All Tags";
+
+    const descEl = document.querySelector('head meta[name="description"]');
+    descEl.setAttribute('content', 'See all our tags');
+
+    const robotEl = document.querySelector('head meta[name="robots"]');
+    robotEl.setAttribute('content', 'noindex, nofollow');
```
Comment (nilmadhab) [inline src/views/TagsList.vue]: why no follow ?

Comment (shivamag00) [reply src/views/TagsList.vue]: This page lists all the tags. We are indexing all the pages of individual tags. Hence, I thought we should not index this page. Should I index this page also?

Comment (nilmadhab) [reply src/views/TagsList.vue]: yes, index it too

Comment (shivamag00) [reply src/views/TagsList.vue]: Done

**Pull request merged**